Docker/docker-compose.yml

```diff
@@ -106,6 +110,8 @@ services:
       - "4000"
       - "4001"
       - "4002"
+    ports:
+      - "7003:4001" # expose the port for the worker to accept transactions from clients
```

Comment:
1. Good catch
2. in general, annotations port number -> purpose of port would be great overall.